index.js

```diff
@@ -1 +1,129 @@
 console.log("Hello!!!");
+
+// Create class UserService
+// Implement every scenario from dummy users BE in your class
+// For each scenario create new method
+// Methods can be static
+// Read response from BE and console.log it or store it in class properties
+// Don't forget to catch errors when request failed
+// *Extra task: Delete users with ids 1,2,3,4 and random id from 5 to 10000 simultaneously
+// Use Promise.allSettled for this task
+
+class UserService {
+    users = [];
+    selectedUser = null;
+    searchedUsers = [];
+    filteredUsers = [];
+    userCarts = null;
+    userPosts = null;
+    userTodos = null;
+    getAllUsers(){
+        fetch("https://dummyjson.com/users")
+        .then((res) => {
+            return res.json()
+        })
+        .then((data) => {
+            this.users.push(...data.users);
+            console.log(this.users)
+        }) 
+    }
+    async getUser(id){
+        const url = new URL(`https://dummyjson.com/users/${id}`);
+        const res = await fetch(url);
+        const data = await res.json();
+        this.selectedUser = data;
+        console.log(this.selectedUser);
+    }
+    searchUsers(param){
```

Comment:
it's a good practice to have one way of handling promises in class, module, function